test/AuthenticationRequest.request.spec.ts

```diff
@@ -381,24 +381,24 @@ describe('create Request JWT should', () => {
         }
       },
       claims: {
-        'id_token': {
-          'acr': null
-        },
-        'vp_token': {
-          'presentation_definition': {
-            'id': 'Insurance Plans',
-            'input_descriptors': [
-              {
-                'id': 'Ontario Health Insurance Plan',
-                'schema': [
-                  {
-                    'uri': 'https://did.itsourweb.org:3000/smartcredential/Ontario-Health-Insurance-Plan'
-                  }
-                ]
-              }
-            ]
+        presentationDefinitions: [
```

Comment:
this is why I didn't change the actual code and just wrote review, in the past two hours I'm trying to find examples about the spec that have this kind of format, but I couldn't find any

Reply:
This is an example of using it in a vp_token: see https://openid.net/specs/openid-connect-4-verifiable-presentations-1_0.html#section-11.6.1.1